steps/wellbeing-support/home-check.tsx

```diff
@@ -0,0 +1,87 @@
+import { FieldsetLegend } from "lbh-frontend-react/components";
+import React from "react";
+import {
+  ComponentDatabaseMap,
+  ComponentWrapper,
+  DynamicComponent,
+  StaticComponent
+} from "remultiform/component-wrapper";
+import { Paragraph } from "lbh-frontend-react/components/typography/Paragraph";
+
+import { makeSubmit } from "../../components/makeSubmit";
+import ProcessStepDefinition from "../../components/ProcessStepDefinition";
+import { RadioButtons } from "../../components/RadioButtons";
+import DatabaseSchema from "../../storage/DatabaseSchema";
+import processRef from "../../storage/processRef";
+
+import PageSlugs, { hrefForSlug } from "../PageSlugs";
+import PageTitles from "../PageTitles";
+
+const step: ProcessStepDefinition = {
+  title: PageTitles.HomeCheck,
+  heading:
+    "Are you doing a Home Check as part of the Tenancy and Household Check?",
+  step: {
+    slug: PageSlugs.HomeCheck,
+    nextSlug: PageSlugs.Health,
+    Submit: makeSubmit({
+      href: hrefForSlug(PageSlugs.Submit),
```

Comment:
This `href` should match `nextSlug`.

Long term, I'm not sure of the utility of `nextSlug`...